source/_components/homekit_controller.markdown

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title: "Homekit controller support"
+description: "Instructions how to integrate your Homekit devices within Home Assistant."
+date: 2018-03-19 21:04
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: homekit.png
```

Comment:
we already have a `apple-homekit.png` resource, I suggest you use that.

Reply:
Done